front/pages/edit.vue

```diff
@@ -307,6 +369,12 @@ export default {
   },
   data() {
     return {
+      showHelp: true,
```

Comment:
Showing that help modal by default is gonna be annoying when going back and forth between edit and index.

Maybe add a little help icon button on the controls panel?